.github/workflows/rust.yml

```diff
@@ -0,0 +1,44 @@
+name: CI
+
+on: [push]
```

Comment:
Don't like the idea to run this on every push. It is better to run only within the Pull Request and on push to `main`. Example here: https://github.com/qoollo/bob/blob/master/.github/workflows/build.yml#L3